src/Movement/Kinematics/HangprinterKinematics.h

```diff
@@ -57,7 +57,8 @@ class HangprinterKinematics : public RoundBedKinematics
 
 private:
 	// Basic facts about movement system
-	static constexpr size_t HANGPRINTER_AXES = 4;
+	const char* ANCHOR_CHARS = "ABCDE";
```

Comment:
I think we discussed anchor chars some other time. The letter here E is confusing here since the extruder axis is already called E

Reply:
I managed to find the previous discussion https://github.com/Duet3D/RepRapFirmware/pull/585#discussion_r1055352961

I'm still in favor of naming the axes ABCDIJKLO. The naming is important since it will face users and will probably go into build manuals and instruction videos. Later firmware implementations will maybe have to imitate our choice.

Reply:
Yes, not only we need to maintain the first ABCD for backwards compatibility with any previous hangprinter, but also we shouldn't conflict with any future notion of extra axes beyond XYZ.
I want to rotate the head because I've seen we can. Not arbitrarily, but 3 extra axes in some angle depnding on how do you build it mechanically. 
That could serve for some tool changing systems, a bit of rotation may be enough.
Also that could serve give stability to alltop configs, possibly allowing them to move faster.
I know, that's more fast forward calculations because now you need to know where the anchors are attached in the head to rotate it precisely. 
In any case, I think that decision was right and I just forgot about it. It's the E, that's for the extruder, even worse.
But thanks for the reminder of the order we decided for anchors: ABCDIJKLO.
That must be fixed, thanks.

Reply:
I'm going to use ABCDIJKLO directly even though that's not needed for 5 anchors because I will likely forget again and the loops accessing ANCHOR_CHARS should be properly limited anyway.
We could add a diff line for every new anchor:

--- ANCHOR_CHARS = "ABCDIJ"
+++ ANCHOR_CHARS = "ABCDIJK"

but I think it's fine to do it all at once. If not, I can easily remove JKLO and re-add it again in latter commits as needed.

Reply:
I think adding all the chars at once is good :+1: